### ISSUE TITLE
Allow the Staker to Include Multiple Claimants per Stake

### ISSUE THREAD

**Issue by eduardo-munoz** [Bounties-Network/Delphi#95]
# Problem
- Only one wallet address can be listed as the claimant within a stake
- Within adChain's "Smart IOs", Advertisers (stakers) will be needing to whitelist multiple Publishers (claimants)
- With the current code, ADVs would have to create individual stakes per PUB (up to 65 claimants since there are 65 PUBs in the adChain Publisher Registry

# Solution
- [ ] Within the `whitelistClaimant` function, add the ability to whitelist multiple claimants in order to minimize the number of unnecessary transactions

**Comment by mbeylin** [comment]
Question: would the batch-whitelisted claimants be treated as a coalition (i.e. they all open claims in tandem), or does it suffice to have a function which allows for bulk whitelisting, but each entry is it's own whitelist, capable of opening independent claims?

**Comment by eduardo-munoz** [comment]
Correct, we see each wallet address having the ability to open independent claims. Since each wallet address will be a Publisher, they will each be claiming unique amounts.

**Comment by skmgoldin** [comment]
I think literally this will suffice, Mark:

```
    function whitelistClaimants(address[] _claimants, uint[] _deadlines)
    public
    onlyStaker
    {
        require(_claimants.length == _deadlines.length);
        for(uint i = 0; i < _claimants.length; i++) {
            whitelistClaimant(_claimants[i], _deadlines[i]);
        }
    }
```

**Comment by mbeylin** [comment]
@skmgoldin will need arrays for all other fields and requirements that all arrays are of equal length, but yes this makes sense.
